src/content/community/releases/_index.en.md

```diff
@@ -7,34 +7,33 @@ weight = 40
 ## v0.9.0
 
 * The gateway Pod has been renamed from `submariner` to `submariner-gateway`.
-* The Helm charts now support Submariner deployments using the Operator.
-* Broker creation is now managed by the Operator (and not `subctl`).
+* The Helm charts now use Submariner's Operator to deploy and manage Submariner.
+* Broker creation is now managed by the Operator instead of `subctl`.
 * Each Submariner Pod now has its own service account with appropriate privileges.
 * For OpenShift, the Lighthouse CoreDNS server metrics are now included with the default CoreDNS metrics.
 * The `submariner_connections` metric is renamed to `submariner_requested_connections`.
-* `service-discovery` flag of `deploy-broker` has been deprecated in favor of `components` flag.
+* The `service-discovery` flag of `subctl deploy-broker` has been deprecated in favor of the `components` flag.
 * For cases in which cross-cluster connectivity is provided without Submariner, `subctl` can now just deploy
-  Service Discovery via the `--components` flag.
-* Improved Service CIDR discovery for k3s deployments.
-* All Submariner metrics are now prefixed with `submariner_` in Prometheus.
+  Service Discovery.
+* Improved Service CIDR discovery for K3s deployments.
+* All Submariner Prometheus metrics are now prefixed with `submariner_`.
 * With Globalnet deployments, Global IPs are now assigned to exported Services only. Previously, Globalnet annotated
   every Service in the cluster, whether or not it was exported.
-* Lighthouse now allows a user to set name of CoreDNS custom ConfigMap.
-* The `strongswan` cable driver which was deprecated in v0.8.0 release is now removed.
+* The name of the CoreDNS custom ConfigMap for service discovery can now be specified on `subctl join`.
+* The `strongswan` cable driver that was deprecated in the v0.8.0 release is now removed.
 * The Lighthouse-specific API is now removed in favor of [Kubernetes Multicluster Services API](https://github.com/kubernetes-sigs/mcs-api/).
-* A new tool [`subctl validate`](https://submariner.io/operations/deployment/subctl/#validate) was added that detects issues with the
+* A new tool, [subctl validate](https://submariner.io/operations/deployment/subctl/#validate), was added that detects issues with the
```

Comment:
Change to diagnose as per https://github.com/submariner-io/submariner-operator/pull/1309